### DISCUSSION TITLE
Any Chance to Unlock the osu!lazer 1000fps Cap?

### DISCUSSION THREAD

**Discussion by suntube** [ppy/osu#26569] [Q&A] [answered]
Hello

I've been playing with osu! for over a decade now, and I recently got my hands on a high-end pc.  I was all set to unleash the full potential of osu!lazer when I stumbled upon a little hiccup – it seems like the FPS is capped at 1000fps, and I hear this might've been a deliberate move by our big boss @peppy .

I totally respect your vision, peppy, but I can't help but wonder if there's a way to go beyond the 1000fps limit.  While it may not be a deal-breaker for most players, for those of us who are obsessed with squeezing out every ounce of performance, it's a real curiosity!

So, I thought I'd drop a post here and see if there's any possibility of unlocking this cap, even if it's just for experimental purposes. 

If not, no worries we are here forever anyway so I will just click circles until it happens in future.

**Comment by bdach** [comment, marked as answer]
No, that won't happen, and there is _no reason_ to do that. Read: https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates
